redwood-treehouse/src/commonMain/kotlin/app/cash/redwood/treehouse/AppLifecycle.kt

```diff
@@ -30,8 +30,16 @@ public interface AppLifecycle : ZiplineService {
   public interface Host : ZiplineService {
     public fun requestFrame()
 
+    /** Notify the host that an event was unrecognized and will be ignored. */
     public fun onUnknownEvent(widgetTag: WidgetTag, tag: EventTag)
 
+    /**
+     * Notify the host that an event was received for a node that either never existed or that no
+     * longer exists. That event will be ignored.
```

Comment:
```suggestion
     * Notify the host that an event was received for a node that no longer exists.
     * That event will be ignored.
```
A node that never existed should never happen if our code is correct, whereas a no-longer-existing node is quite plausible.